kong/vault_service.go

```diff
@@ -152,3 +156,28 @@ func (s *VaultService) ListAll(ctx context.Context) ([]*Vault, error) {
 	}
 	return vaults, nil
 }
+
+func (s *VaultService) Validate(ctx context.Context, vault *Vault) (bool, string, error) {
```

Comment:
Could it be written like a Validate [here](https://github.com/Kong/go-kong/blob/abc671e70baee66723ed806889930d07b65f2a67/kong/route_service.go#L216C1-L230)? I think it's more consistent

Reply:
👍 Although I'd mention what the return values are. One could deduce what the `bool` is for but string isn't obvious here IMHO.

Reply:
Yup it ought to be explained in docstring

Reply:
Done.